intrahospital_api/apis/prod_api.py

```diff
@@ -146,6 +143,20 @@ def get_or_fallback(self, primary_field, secondary_field):
 
         return result
 
+    def get_identifier(self):
+        """ for reasons unknown the upstream database often has the
```

Comment:
It would be interesting for this to be less unkonwn... :)

Reply:
I was assuming it was for patients not at the RFH, ie liaison, but some investigation could be done.